app/models/comment.rb

```diff
@@ -272,128 +272,128 @@ def record_wrangling_activity
+      edited_feedback.update_attribute(:read, false)
+    else # original inbox comment was deleted
+      add_feedback_to_inbox(user)
     end
+  end
 
-    def content_too_different?(new_content, old_content, threshold)
-      # we added more than the threshold # of chars, just return
-      return true if new_content.length > (old_content.length + threshold)
-
-      # quick and dirty iteration to compare the two strings
-      cost = 0
-      new_i = 0
-      old_i = 0
-      while new_i < new_content.length && old_i < old_content.length
-        if new_content[new_i] == old_content[old_i]
-          new_i += 1
-          old_i += 1
-          next
-        end
+  def add_feedback_to_inbox(user)
+    new_feedback = user.inbox_comments.build
+    new_feedback.feedback_comment_id = self.id
+    new_feedback.save
+  end
 
-        cost += 1
-        # interrupt as soon as we have changed > threshold chars
-        return true if cost > threshold
+  def content_too_different?(new_content, old_content, threshold)
+    # we added more than the threshold # of chars, just return
+    return true if new_content.length > (old_content.length + threshold)
 
-        # peek ahead to see if we can catch up on either side eg if a letter has been inserted/deleted
-        if new_content[new_i + 1] == old_content[old_i]
-          new_i += 1
-        elsif new_content[new_i] == old_content[old_i + 1]
-          old_i += 1
-        else
-          # just keep going
-          new_i += 1
-          old_i += 1
-        end
+    # quick and dirty iteration to compare the two strings
+    cost = 0
+    new_i = 0
+    old_i = 0
+    while new_i < new_content.length && old_i < old_content.length
+      if new_content[new_i] == old_content[old_i]
+        new_i += 1
+        old_i += 1
+        next
       end
 
-      cost > threshold
-    end
+      cost += 1
+      # interrupt as soon as we have changed > threshold chars
+      return true if cost > threshold
 
-    def not_user_commenter?(parent_comment)
-      (!parent_comment.comment_owner && parent_comment.comment_owner_email && parent_comment.comment_owner_name)
+      # peek ahead to see if we can catch up on either side eg if a letter has been inserted/deleted
+      if new_content[new_i + 1] == old_content[old_i]
+        new_i += 1
+      elsif new_content[new_i] == old_content[old_i + 1]
+        old_i += 1
+      else
+        # just keep going
+        new_i += 1
+        old_i += 1
+      end
     end
 
-    def have_different_owner?(parent_comment)
-      return not_user_commenter?(parent_comment) || (parent_comment.comment_owner != self.comment_owner)
+    cost > threshold
+  end
+
+  def not_user_commenter?(parent_comment)
+    (!parent_comment.comment_owner && parent_comment.comment_owner_email && parent_comment.comment_owner_name)
+  end
+
+  def different_owner?(parent_comment)
+    not_user_commenter?(parent_comment) || (parent_comment.comment_owner != self.comment_owner)
+  end
+
```

Comment:
No change above this point except for indentation being fixed and one function renamed to please the hound.